internal/operators/ocs/custom_manifest.go

```diff
@@ -0,0 +1,82 @@
+package ocs
+
+import (
+	"bytes"
+	"os"
+	"text/template"
+)
+
+// Subscription Manifest for testing internal builds
+func ocsCustomSubscription() (string, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":         Operator.Namespace,
+		"OPERATOR_SUBSCRIPTION_NAME": Operator.SubscriptionName,
+	}
+
+	const ocsSubscription = `apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "{{.OPERATOR_SUBSCRIPTION_NAME}}"
+  namespace: "{{.OPERATOR_NAMESPACE}}"
+spec:
+  installPlanApproval: Automatic
+  name: ocs-operator
```

Comment:
How the channel is picked up @priyanka19-98 ? Or if it's not specified it will take the default which is defined as the default channel in the packageManifest?  Or is there any other logic later on which is updating channel in the subscription?

Reply:
default channel from the packageManifest is taken.

Reply:
do we need a way to specify channel as well?

Reply:
We are always specifying in our automation so I am not sure how it works if not defined as I haven't tried.
Have you tried to verify?

Reply:
yes, we don't specify a channel and it takes the latest channel always. But for testing I guess we would need to specify a channel. What should be the default channel?

Reply:
Depends on version. It's stable-x.y where x.y is installed version of ocs. So if you will use image for 4.8 then stable-4.8 . If testing 4.7 build, then stable-4.7.